src/traces/scattercarpet/hover.js

```diff
@@ -67,7 +67,7 @@ module.exports = function hoverPoints(pointData, xval, yval, hovermode) {
     var tj = ij[1] - j0;
 
     var xy = carpet.evalxy([], i0, j0, ti, tj);
-    text.push('y: ' + xy[1].toFixed(3));
+    text.push('y = ' + xy[1].toFixed(3));
```

Comment:
cc @rreusser unless that `y:` was on purpose? This here turns it into `y =` just like the `a =` and `b =` rows

![peek 2017-10-03 17-01](https://user-images.githubusercontent.com/6675409/31149016-7e8e0fa0-a85c-11e7-9ff3-127967dc7c27.gif)

Reply:
Most other places (heatmap/contour, 3d, ternary) we use `:` - perhaps we should standardize on that here too?

Reply:
Good catch. Thank you. That seems correct.

Reply:
Oh. Haha, my mistake then. Perhaps it's the `a/b = ` that are the odd ones out. Need to check the consistent choice here.

Reply:
https://github.com/plotly/plotly.js/blob/master/src/components/fx/hover.js#L697-L698

Reply:
Sorry, to be less vague, `:` seems correct. `=` seems incorrect.

Reply:
done in 2fe641d